authors/apps/comments/utils/model_helpers.py

```diff
@@ -52,3 +53,159 @@ def get_comment_history(comment):
         }
         edit_history_list.append(edit_history_info)
     return edit_history_list
+
+
+def check_if_user_is_object_author(obj, user):
+    return obj.author.user == user
+
+
+class LikeDislikeObject():
```

Comment:
Would it also be possible to add a few more comments to this class? It is quite lengthy and feels sort of abstract.